packages/grant-explorer/src/test-utils.tsx

```diff
@@ -100,8 +100,8 @@ const makeTimestamp = (days?: number) =>
   Math.floor(Number(faker.date.soon(days)) / 1000).toString();
 
 export const makeRoundMetadata = (
-  overrides?: Partial<__deprecated_RoundMetadata>
-): __deprecated_RoundMetadata => ({
+  overrides?: Partial<any>
+): any => ({
```

Comment:
can we use type `Round` here?

Reply:
let's not use `any`, if it's deprecated but still use let's keep it or create a new type that we will continue to use

Reply:
@gravityblast / @0xKurt lol okay i was using it only for test but this makes sense. Fixed